index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.2.0/css/all.css"
+      integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="CSS/styles.css" />
+    <link rel="stylesheet" href="CSS/main.css" />
+    <title>Natnael-Portfolio</title>
+  </head>
+  <body>
+    <section class="container">
+      <nav class="normal-button">
+        <div class="logo">
+          <div class="placeholder">
+            <div><a href="/"></a></div>
+            <div>N-A</div>
+          </div>
+        </div>
+
+        <button type="button" class="hamburger" id="hamburger">
+          <i class="fas fa-bars"></i>
+        </button>
+        <button type="button" id="close" >
+          <i class="fas fa-times"></i>
+        </button>
+        <ul class="nav-ul">
+          <li><a href="#portfolio" class="nav-text-1">Portfolio</a></li>
+          <li><a href="#about" class="nav-text-2">About</a></li>
+          <li><a href="#contact" class="nav-text-3">Contact</a></li>
+        </ul>
+      </nav>
+
+      <section class="headline" id="portfolio">
```

Comment:
This section is the headline and not the portfolio per see. 
When the user clicks on the portfolio button, needs to be directed to your work section, which needs to have the projects you built before. 
As we have to respect the Figma design, we need a work section in our portfolio. Kindly, add back the cards with your work history.
This one is the work section that we need to have on the portfolio in accordance with the Figma design chosen by you:
Desktop:
![image](https://user-images.githubusercontent.com/83596135/169075990-1918f524-01ed-412f-a456-1f0e89679f14.png)
Mobile:
![image](https://user-images.githubusercontent.com/83596135/169076195-82700f3e-d5fb-46ad-b1ce-98095d149e1e.png)


- [ ] Kindly, implement back on your portfolio the work section with all the projects as in the Figma design :+1: